frontend_tests/.eslintrc.json

```diff
@@ -1,7 +1,9 @@
 {
     "env": {
-        "shared-node-browser": true
+        "shared-node-browser": true,
+        "es6": true
```

Comment:
This should be in another commit.

Reply:
It can't be since, this commit will fail linting, spread operator had another rule then it got desperate. I think I will update it in commit msg to explain this better.

Reply:
Yeah, the previous commit also will also work, it was just that I made those changes and then figured this out so, I didn't feel like making a new one.

Reply:
I see. I mean making a new commit and moving around commits is all worth the clarity.